test/e2e/page-objects/pages/token-overview-page.ts

```diff
@@ -0,0 +1,54 @@
+import { Driver } from '../../webdriver/driver';
+
+class TokenOverviewPage {
+  private driver: Driver;
+
+  private readonly sendButton = {
+    text: 'Send',
+    css: '.icon-button',
+  };
+
+  private readonly receiveButton = {
+    text: 'Receive',
+    css: '.icon-button',
+  };
+
+  private readonly swapButton = {
+    text: 'Swap',
+    css: '.icon-button',
```

Comment:
A couple of small things:
- could we sort these selectors alphabetically maybe?
- I think these buttons are the exact same buttons we have in the home screen. Maybe we could separate them in their own component, and re-use them in  the homepage class and in the token overview class? Just a thought, it doesn't have to be in this PR. The advantage would be that if a button from this group changes, we'll only need to update it in one place. The disadvantage might be that it might complicate a bit the component? though not too much. What are your thoughts? In any case, it can be left outside this PR, if we decide to have this separate component

![Screenshot from 2024-12-20 10-45-01](https://github.com/user-attachments/assets/b2035526-778c-4ef7-be23-3910e7dbcf3a)

![Screenshot from 2024-12-20 10-45-09](https://github.com/user-attachments/assets/df86e867-f929-4cda-a178-ad910532c846)

Reply:
Yes, I also had the same thought. It could make sense (if we keep it text based as I have done here) 
the only doubt that I have is the depending on the token type different buttons are displayed so that is still something to think about. I do agree though there is a way we could make this easier to deal with